### ISSUE TITLE
Gmail Redesign

### ISSUE THREAD

**Issue by NTICompass** [mailvelope/mailvelope#591]
I'm using Mailvelope 2.2.2 on Firefox 61 (Developer Edition) and I got an option in Gmail to switch to "new Gmail" (https://gsuiteupdates.googleblog.com/2018/06/new-gmail-timeline.html).

The "classic Gmail" worked with Mailvelope, but with the "new Gmail" nothing happens and I just see emails with 2 attachments - noname and encrypted.asc.

**Comment by NTICompass** [comment]
Interestingly, if I click on encrypted.asc, Gmail shows the attachment as text and then Mailvelope is able to decrypt it.

**Comment by BlackDex** [comment]
I experience the same.

**Comment by My1** [comment]
but more or less recently, the encrypted.asc I get from Facebook just leaves a white output window.

**Comment by My1** [comment]
ouch I havent seen that message. but maybe the stored key is broken. try to reimport it.

**Comment by Navneet-Suresh** [comment]
ok, I will update my keys and report back.

**Comment by My1** [comment]
is there anything "special" about this key? maybe a curve that isnt RSA or some relatively new or uncommon way of encryption?

**Comment by Navneet-Suresh** [comment]
Nope, You can import [my pub key](https://pgp.mit.edu/pks/lookup?op=get&search=0xF47005CF2F5F04A6) and have a look.
